.github/workflows/ci.yml

```
@@ -39,7 +42,8 @@ jobs:
         with:
           node-version: 20
       - run: npm ci
-      - run: npm test
+      - name: Test coverage
+        uses: ArtiomTr/jest-coverage-report-action@v2
```

Please pin this external project to a specific revision:
```suggestion
        uses: ArtiomTr/jest-coverage-report-action@262a7bb0b20c4d1d6b6b026af0f008f78da72788
```

I think you will also need to set a github token to use this in the org.